python/tests/test_lowlevel.py

```diff
@@ -1832,7 +1838,7 @@ def f(mutations):
         u = ts.get_num_nodes()
         for bad_node in [u, u + 1, 2 * u]:
             self.assertRaises(_tskit.LibraryError, f, [(0.1, bad_node)])
-        for bad_pos in [-1, length, length + 1]:
+        for _bad_pos in [-1, length, length + 1]:
```

Comment:
Error here, length should be bad_pos

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/479/commits/0900c826fb1c821ef35e87784bae4f94b895f8f9